docs/reference/search/search-your-data/learning-to-rank-model-training.asciidoc

```diff
@@ -0,0 +1,150 @@
+[[learning-to-rank-model-training]]
+=== Deploy and manage Learning To Rank models
+++++
+<titleabbrev>Deploy and manage LTR models</titleabbrev>
+++++
+
+preview::["The Learning To Rank feature is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but this feature is not subject to the support SLA of official GA features."]
+
+NOTE: This feature is available for Elastic Stack versions 8.12.0 and newer and requires a https://www.elastic.co/pricing[Platinum subscription] or higher.
+
+[discrete]
+[[learning-to-rank-model-training-workflow]]
+==== Train and deploy a model using Eland
+
+https://xgboost.readthedocs.io/en/stable/[XGBoost^] model training typically leverages a standard, Python data science toolkit such as Pandas and scikit-learn
+
+We have developed an example notebook available https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/08-learning-to-rank.ipynb[here^], detailing an end-to-end model training and deployment workflow.
+
+We highly recommend integrating https://eland.readthedocs.io/[eland^] into your workflow since it provides some important features in the workflow to integrate Learning To Rank in {es}:
+
+* Configure feature extraction
+
+* Extract features for training
+
+* Deploy the model into {es}
+
+[discrete]
+[[learning-to-rank-model-training-feature-definition]]
+===== Configure feature extraction in Eland
+
+Feature extractors are defined using templated queries. https://eland.readthedocs.io/[Eland^] provides the `eland.ml.ltr.QueryFeatureExtractor` to define these feature extractors directly in Python:
```

Comment:
If we want to backport to 8.12 it would make sense for the Eland docs to be up to date, otherwise users won't find any LTR info there. cc @pquentin

Reply:
I would like to be able to publish this guide for 8.12 and fix the eland doc in parallel. @pquentin is it possible?

Reply:
Another option is to merge to 8.13 and main, and wait to manually backport to 8.12 until those Eland docs are up to date

Reply:
I would prefer to merge right now to 8.12.0 because some customer of us are currently invited to test it.